src/tiledbsoma_ml/gpu_batches.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2021-2024 The Chan Zuckerberg Initiative Foundation
```

Comment:
other repos are moving to the model of omitting dates from copyright headers.  Suggest doing the same in this repo.

Reply:
works for me